src/urdf_parser_py/display_urdf.py

```diff
@@ -0,0 +1,20 @@
+import sys
+import argparse
+
+from urdf_parser_py.urdf import URDF
+
+
+def main(argv=None):
+    parser = argparse.ArgumentParser(argv, usage='Load an URDF file')
```

Comment:
I'd be surprised if argv is ever not `None`. This happens to be ok because it looks like the first argument is `prog` which is supposed to be the name of the program, else `sys.argv[0]` if `None`. https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser

Still I think this should be removed for clarity.